modules/user_manual/pages/files/media_viewer_app.adoc

```diff
@@ -0,0 +1,86 @@
+= The Media Viewer App
+:browser-image-support-url: https://en.wikipedia.org/wiki/Comparison_of_web_browsers#Image_format_support
+:webm-url: https://www.webmproject.org/
+:ogg-url: https://xiph.org/vorbis/
+:mp4-url: https://en.wikipedia.org/wiki/MPEG-4_Part_14
+:media-viewer-app-url: https://marketplace.owncloud.com/apps/files_mediaviewer
+
+image:media-viewer-app/app-overview.gif[]
+
+== Introduction
+
+The {media-viewer-app-url}[Media Viewer app] is a lightweight viewer for pictures and videos which integrates with the files app, and is released under the GPLv2. 
+It replaces the gallery and files_videoplayer apps, which have now been deprecated, and supports the same basic feature set as the deprecated apps.
+
+IMPORTANT: Please remove the gallery and files_videoplayer apps before installing it.
+
+It supports the following functionality:
+
+* *Image*: _preview_, _zoom_, _rotation_, and _download_
+* *Video*: _playback_ and _download_
+
+NOTE: The app will support paginating through all media files in the current directory, even if only one media file was chosen to be previewed.
+
+== Features
+
+* Support for a large selection of image and video formats (depending on server setup)
+* Fullscreen, zoomable slideshow view integrated with the Files view and Public Links
+* Image rotation
+* Sort images by name or date
+* Image and video download straight from the slideshow
+* Native SVG support
+* Mobile support
+
+== Supported File Formats
+
+* The supported video formats depend on the user's browser
+* The supported image formats depend on the server capabilities
+
+== Configuring the Media Viewer App
```

Comment:
Everything below is actually admin doc, but this file is part of the user documentation